src/ts/index.ts

```diff
@@ -0,0 +1,152 @@
+class parsedInput {
+  num1: string;
+  num2: string;
+  operator: string;
+
+  constructor(operator: string, num1: string, num2: string) {
+    this.operator = operator;
+    this.num1 = num1;
+    this.num2 = num2;
+  }
+}
+
+function findOperator(str: string): string {
+  if (str.indexOf("/") != -1) {
+    return "/";
+  } else if (str.indexOf("X") != -1) {
+    return "X";
+  } else if (str.indexOf("-") != -1) {
+    return "-";
+  } else if (str.indexOf("+") != -1) {
+    return "+";
+  } else {
+    return "none";
+  }
+}
+
+function parseInput(str: string): parsedInput {
+  const total = document.getElementById("total") as HTMLElement;
+  let prompt = total.innerHTML as string;
```

Comment:
```suggestion
  const total: HTMLElement = document.getElementById("total") as HTMLElement;
  let prompt: string = total.innerHTML as string;
```

저는 적는게 좋다고 생각하는데 혹시 어떻게 생각하시나요?

Reply:
저도 빠뜨린 부분이 있는데 적어주는게 좋다고 생각합니다.

Reply:
@hochan222 @jwon42 뒤에 강제 타입캐스팅을 해놓더라도 가독성 측면에서 앞에 있는 게 좋다는 의미죠?

Reply:
둘이 다른 의미라고 생각합니다. c 언어로 치자면

```c
HTMLElement total = (HTMLElement)document.getElementById("total");
```

따라서, total 뒤에 붙는 HTMLElement는 형변환 요소가 아닌 변수 선언의 Type의미를 가져서 생략하면 안된다고 생각합니다.